metadata/Cargo.toml

```diff
@@ -14,7 +14,9 @@ homepage.workspace = true
 description = "Command line utilities for checking metadata compatibility between nodes."
 
 [dependencies]
-codec = { package = "parity-scale-codec", workspace = true, features = ["derive"] }
+codec = { package = "parity-scale-codec", workspace = true, features = [
+    "derive",
```

Comment:
dq: I do like the prior formatting more since I find one-liners easier to follow. Did you find any good formatters for .toml or its just personal preference?

Reply:
Oh, I use the [`Even Better TOML`](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) VSCode extension for formatting. Let me revert it.

Reply:
@tadeohepperle can you fix this as well?